### ISSUE TITLE
S390x images won't boot

### ISSUE THREAD

**Issue by kingsleyzissou** [achilleas-k/osbuild-devel#3]
I couldn't get any of the s390x guest images to boot properly with the `./vm/start` script. They images didn't have any network activity at all... I think it was something to do with the PCI network device. I changed it to `virtio-net-ccw` and all is good now.

I'm not sure if you were having the same issues, but if you were I could put a PR in for that.

**Comment by achilleas-k** [comment]
You're right though the `virtio-net-ccw` network device didn't seem to work for me.  I wonder if it ever worked for me now that you bring it up.  I'll keep the issue open and come back to it when I need it again.
